indexd/drs/blueprint.py

```diff
@@ -1,14 +1,65 @@
+import os
 import flask
 import json
 from indexd.errors import AuthError, AuthzError
 from indexd.errors import UserError
 from indexd.index.errors import NoRecordFound as IndexNoRecordFound
 from indexd.errors import IndexdUnexpectedError
+from indexd.utils import drs_service_info_id_url_reversal
 
 blueprint = flask.Blueprint("drs", __name__)
 
 blueprint.config = dict()
 blueprint.index_driver = None
+blueprint.dist = []
+
+
+@blueprint.route("/ga4gh/drs/v1/service-info", methods=["GET"])
+def get_drs_service_info():
+    """
+    Returns DRS compliant service information
+    """
+    drs_dist = {}
+
+    reverse_domain_name = drs_service_info_id_url_reversal(url=os.environ["HOSTNAME"])
+
+    if len(blueprint.dist) > 0:
+        # Check to see if the information is of type drs. If not, use the available information to return DRS compliant service information
+        for dist in blueprint.dist:
+            if (
+                "type" in dist
+                and isinstance(dist["type"], dict)
+                and "artifact" in dist["type"]
+                and dist["type"]["artifact"] == "drs"
+            ):
+                drs_dist = dist
+        if drs_dist == {}:
+            drs_dist = blueprint.dist[0]
```

Comment:
This doesn't seem right, the first "dist" will probably be a different instance of indexd 🤔